pkgs/development/python-modules/hastygram/default.nix

```
@@ -0,0 +1,81 @@
+{ stdenv, pkgs, fetchFromGitHub, fetchpatch, makeWrapper
+, inkscape, nodejs, mkYarnPackage, python3, python3Packages }:
+
+let
+  pname = "hastygram";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "zopieux";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "11ql00fhjkai7mb1gr22nglca5y2mnlzjig74xadwcjgqdi16v6f";
+  };
+
+  frontend = mkYarnPackage {
+    name = pname;
```

```suggestion
    name = "${pname}-${version};
```

If mkYarnPackage supports pname + version please inherit them.

---

It does, fixed, thanks.